bin/yamlbot

```diff
@@ -26,6 +26,10 @@ parser = OptionParser.new do |opts|
     options[:file] = file
   end
 
+  opts.on('-c', '--color', 'Enable colored output') do |color|
+    options[:color] = color
```

Comment:
I wonder if you shouldn't make color the default?  i.e.

```ruby
options[:color] = not color
```

It's up to you as to what you want default.  I imagine it would be most useful depending on how you intend it to be used most often.  e.g. `--disable-color` and `-c` as the option.